book/src/cross-compiling.md

```diff
@@ -0,0 +1,41 @@
+# Cross-compiling
+
+Lighthouse supports cross-compiling, allowing users to run a binary one
+platform (e.g., `aarch64`) that was compiled on another platform (e.g.,
+`x86_64`).
+
+
+## Instructions
+
+Cross-compiling requires [`Docker`](https://docs.docker.com/engine/install/),
+[`rustembedded/cross`](https://github.com/rust-embedded/cross) and for the
+current user to be in the `docker` group.
+
+The binaries will be created in the `target/` directory of the Lighthouse
+project.
+
+### Targets
+
+The `Makefile` in the project contains four targets for cross-compiling:
+
+- `build-x86_64`: builds an optimized version for x86_64 processors (suitable
+	for most users).
+- `build-x86_64-portable`: builds a version x86_64 processors which avoids
+	using some modern CPU instructions that might cause an "illegal
+	instruction" error on older CPUs.
```

Comment:
There's still an issue with pre-SSSE3 CPUs. I've opened https://github.com/sigp/lighthouse/issues/1504 to track it